components/FilterDropdownSingle/styles.ts

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+
+export const FilterDropdownInput = styled.select<{ $hasValue: boolean }>`
```

Comment:
note: we eventually want to import a custom dropdown arrow icon to ensure that the positioning matches the design